bucketers.py

```diff
@@ -183,8 +214,9 @@ def __init__(self,
             word, freq = line.strip().split('\t')
             freq_counts[word] = freq
       elif freq_corpus_file:
-        for word in itertools.chain(corpus_utils.iterate_tokens(freq_corpus_file)):
-          freq_counts[word] += 1
+        for words in corpus_utils.iterate_tokens(freq_corpus_file):
```

Comment:
Hmm, it seemed to be working fine for me... Maybe this is a Python version dependent issue?